.eslintignore

```diff
@@ -0,0 +1,3 @@
+.eslintrc.js
+src/tests/utils/test-utils.jsx
```

Comment:
I've tried making this a tsx file more than once, but haven't yet found the solution to do so. This file will likely not change, so instead of adding several rules to ignore, I've ignored the file.

Reply:
failing rules were:
```
'children' is missing in props validation               react/prop-types
Unsafe assignment of an `any` value        @typescript-eslint/no-unsafe-assignment
Unsafe argument of type `any` assigned to a parameter of type `ReactElement<any, string | JSXElementConstructor<any>>`  @typescript-eslint/no-unsafe-argument
'@testing-library/react' should be listed in the project's dependencies, not devDependencies import/no-extraneous-dependencies
 ```